### DISCUSSION TITLE
Enable more debug logs for mitmweb

### DISCUSSION THREAD

**Discussion by levyitay** [mitmproxy/mitmproxy#6530] [Q&A]
Hi all, 
I was wondering if there is a way to enable more debug logs for mitmweb. 
specifically seeing Tornado webserver logs

**Comment by mhils** [comment]
Logging is a bit hardcoded here at the moment: https://github.com/mitmproxy/mitmproxy/blob/10.1.5/mitmproxy/tools/main.py#L57-L63

But you should be able to write an addon that simply overrides that to customize log levels on start. :)
